cfgov/v1/wagtail_hooks.py

```diff
@@ -33,3 +36,42 @@ def check_request_site(page, request, serve_args, serve_kwargs):
         if isinstance(page, CFGOVPage):
             if not page.shared:
                 raise Http404
+
+
+PAGES_JSON = settings.PROJECT_ROOT.child('v1', '_generated') + '/pages.json'
+
+def update_key_pages_json(request, page):
+    page = page.specific
+    data = json.load(open(PAGES_JSON))
```

Comment:
Why is the page loading the existing state from the JSON? Why not write this file anew each time save is hit? Then we can gitignore this file.